.specify/scripts/bash/update-agent-context.sh

```
@@ -0,0 +1,719 @@
+set -o pipefail
+
+#==============================================================================
+# Configuration and Global Variables
+#==============================================================================
+
+# Get script directory and load common functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+
+# Get all paths and variables from common functions
+eval $(get_feature_paths)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)